mgmtfn/k8splugin/README.md

```diff
@@ -0,0 +1,311 @@
+# Contiv Networking for Kubernetes
+
+Contiv is integrated with Kubernetes via a CNI plugin. This integration enables you
+to set up and manage inter-pod connectivity in a Kubernetes cluster using Contiv 
+policy model.
+
+## Getting Started
+
+This step-by-step procedure will guide you through a minimal experience of creating
+a Kubernetes cluster with Contiv networking and applying policy between pods.
+
+#### Step 1: Clone and build kubernetes
+
+```
+$ mkdir ~/go/src/github.com/k8s
+$ cd ~/go/src/github.com/k8s
+$ git clone https://github.com/kubernetes/kubernetes
+$ cd kubernetes; make
+```
+
+Note: If the build fails, please retry. It will usually build the binaries on 2nd attempt
+
+#### Step 2: Clone contrib repo
+
+```
+$ cd ~/go/src/github.com/k8s
+$ git clone https://github.com/jojimt/contrib -b contiv
+```
+
+#### Step 3: Clone and build netplugin
+
+```
+$ cd ~/go/src/github.com/k8s
+$ git clone https://github.com/contiv/netplugin
+
+Note:
+*Until the PR is merged, you need to use the following instead*
+*$ git clone https://github.com/jojimt/netplugin -b k8s-vagrant*
+
```

Comment:
Which merge is this dependent on? merge into netplugin, or k8s merge? If it is former, can we avoid this `Note` all together?